pycparser/c_ast.py

```diff
@@ -795,3 +795,15 @@ def children(self):
 
     attr_names = ()
 
+class Pragma(Node):
```

Comment:
Yes, I forgot to add the changed _c_ast.cfg file to the commit. Added in 95ee04fffc2530a7f9966b517d5d9173e91e5b0e